librz/analysis/p/analysis_amd29k.c

```diff
@@ -1,4 +1,4 @@
-// SPDX-FileCopyrightText: 2013-2020 pancake <pancake@nopcode.org>
+// SPDX-FileCopyrightText: 2019 deroad <wargio@libero.it>
```

Comment:
For anyone wondering if this fix is legitimate - @wargio is the original author of the plugin, and copyright was added by a mistake, see https://github.com/radareorg/radare2/commit/d33786799938edb8902150c0881b6ba2f31ec18f#diff-959fa8d64e4155a493eed1d5caa67eaa2b3b82c5978695864f309c7f0bb2315b

Reply:
You are so fast.